src/test/java/Tests/DashboardTests.java

```diff
@@ -0,0 +1,98 @@
+package Tests;
+import org.testng.annotations.Test;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.testng.annotations.BeforeClass;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import pages.DashboardPage;
+import pages.HomePage;
+import pages.LoginPage;
+import Utils.DriverSetup;
+
+@Test
+public class DashboardTests extends DriverSetup{
+	private static final WebDriver WebDriverRefrence = null;
+	LoginPage loginPage ;
+	HomePage homePage;
+	DashboardPage dashboardPage;
+
+	
+	@BeforeClass
+	public void driversetup() {
+		loginPage = new LoginPage(driver);
+		homePage = new HomePage(driver);
+		dashboardPage=new DashboardPage(driver);
+		}
+
+	 @Test(priority=0, description = "Capture Games Played by User")
+		public void CaptureGamesPlayed() throws Exception {
+		 loginPage.login("usa@mailinator.com","rashisingh");
+		 Thread.sleep(3000);
+		 dashboardPage.DashboardButton();
+		 Thread.sleep(3000);
+		 dashboardPage.getgamesplayed();
+	 }
+
+    @Test(priority=1, description = "Verify Single Player Game")
+	public void VerifySinglePlayerGame() throws Exception {
+		dashboardPage.PlayOnFirstTopic();
+		Thread.sleep(3000);
+		dashboardPage.ClickSinglePlayerButton();
+		dashboardPage.playgames();
+		Thread.sleep(6000);	
```

Comment:
use explicit wait in place of thread.sleep at all places

Reply:
Done